```
ERROR: corehq.apps.commtrack.tests.test_xml:CommTrackOTATest.test_force_consumption
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 267, in inner
    return helper(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 238, in __call__
    call_with_settings(fn_with_pre_and_post, run_config.settings, args, kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 253, in call_with_settings
    fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 234, in fn_with_pre_and_post
    self.fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/commtrack/tests/test_xml.py", line 186, in test_force_consumption
    balance_blocks = _get_ota_balance_blocks(self.domain, self.user)
  File "/mnt/commcare-hq-ro/corehq/apps/commtrack/tests/test_xml.py", line 725, in _get_ota_balance_blocks
    return util.extract_balance_xml(restore_config.get_payload().as_string())
  File "corehq/ex-submodules/casexml/apps/phone/restore.py", line 703, in get_payload
    response = self.generate_payload()
  File "corehq/ex-submodules/casexml/apps/phone/restore.py", line 709, in generate_payload
    response = self._generate_restore_response(async_task=async_task)
  File "corehq/ex-submodules/casexml/apps/phone/restore.py", line 758, in _generate_restore_response
    for element in provider.get_elements(self.restore_state):
  File "corehq/ex-submodules/casexml/apps/phone/data_providers/standard.py", line 61, in get_elements
    elements = provider(restore_state)
  File "/mnt/commcare-hq-ro/corehq/apps/fixtures/fixturegenerators.py", line 57, in __call__
    items = FixtureDataItem.by_data_types(restore_user.domain, global_types)
  File "/mnt/commcare-hq-ro/corehq/apps/fixtures/models.py", line 417, in by_data_types
    return get_fixture_items_for_data_types(domain, data_type_ids, bypass_cache)
  File "/vendor/lib/python2.7/site-packages/quickcache/quickcache.py", line 40, in inner
    return helper(*args, **kwargs)
  File "/vendor/lib/python2.7/site-packages/quickcache/quickcache_helper.py", line 145, in __call__
    return self.call(*args, **kwargs)
  File "/vendor/lib/python2.7/site-packages/quickcache/quickcache_helper.py", line 55, in call
    key = self.get_cache_key(*args, **kwargs)
  File "/vendor/lib/python2.7/site-packages/quickcache/quickcache_helper.py", line 126, in get_cache_key
    for value in values)
  File "/vendor/lib/python2.7/site-packages/quickcache/quickcache_helper.py", line 126, in <genexpr>
    for value in values)
  File "/vendor/lib/python2.7/site-packages/quickcache/quickcache_helper.py", line 112, in _serialize_for_key
    raise ValueError('Bad type "{}": {}'.format(type(value), value))
ValueError: Bad type "<type 'frozenset'>": frozenset([])
```

--- a/corehq/apps/fixtures/models.py
+++ b/corehq/apps/fixtures/models.py
@@ -413,7 +413,7 @@ class FixtureDataItem(Document):
 
     @classmethod
     def by_data_types(cls, domain, data_types, bypass_cache=False):
-        data_type_ids = frozenset(_id_from_doc(d) for d in data_types)
+        data_type_ids = set(_id_from_doc(d) for d in data_types)
         return get_fixture_items_for_data_types(domain, data_type_ids, bypass_cache)
 
     @classmethod

